### DISCUSSION TITLE
Scaling Strategy

### DISCUSSION THREAD

**Discussion by ChayanBansal** [kedacore/keda-docs#526] [General]
Can someone explain how the custom scaling strategy and works and what the following parameters actually mean?

```
customScalingQueueLengthDeduction: 1      # Optional. A parameter to optimize custom ScalingStrategy.
customScalingRunningJobPercentage: "0.5"  # Optional. A parameter to optimize custom ScalingStrategy.
```

The doc has little information about the scaling strategy so it would be great if someone can explain them here. Also the https://github.com/kedacore/keda/pull/1227 PR does not clear this in detail.

**Comment by zroubalik** [comment]
@TsuyoshiUshio ^^

**Comment by JCMais** [comment]
as someone just looking at this today, it is still not clear what is the meaning of those
